source/components/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TextInputProps } from 'react-native';
+import styled, { useTheme } from 'styled-components/native';
+import Text from '../Text';
+
+type InputProps = Omit<TextInputProps, 'onBlur'> & {
+  onBlur: (value: string) => void;
+  center?: boolean;
+  transparent?: boolean;
+  colorSchema?: 'blue' | 'green' | 'red' | 'purple';
+  showErrorMessage?: boolean;
+  error?: { isValid: boolean; message: string };
+  textAlign?: 'left' | 'center' | 'right';
+};
+
+const StyledTextInput = styled.TextInput<InputProps>`
+  width: 100%;
+  font-weight: ${({ theme }) => theme.fontWeights[1]}
+  background-color: ${({ theme, colorSchema, transparent }) =>
+    transparent ? 'transparent' : theme.colors.complementary[colorSchema][2]};
+  border-radius: 4.5px;
+  border: solid 1px
+  ${({ theme, error }) =>
+    error?.isValid || error === undefined ? 'transparent' : theme.colors.primary.red[0]};
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin: 3px;
+  color: ${({ theme }) => theme.colors.neutrals[0]};
+  ${props => (props.center ? 'text-align: center;' : null)}
+`;
+
+const StyledErrorText = styled(Text)`
+  font-family: Roboto;
```

Comment:
Use font family from theme

Reply:
Actually not needed at all, since I'm styling our Text component to start with, so removed.